File: easyapi_java/src/main/java/com/hlab/easyapi_java/easyapi/util/LoaderInterface.java
package com.hlab.easyapi_java.easyapi.util;

/**
 * Created by H.T. on 05/01/18.
 */

public interface LoaderInterface {
  void showLoading();

  void hideLoading();

  void showNoInternet();

  void showError(String message);
}

File: easyapi_kotlin/src/main/java/com/hlab/easyapi_kotlin/easyapi/util/LoaderInterface.kt
package com.hlab.easyapi_kotlin.easyapi.util

/**
 * Created by H.T. on 05/01/18.
 */

interface LoaderInterface {
  fun showLoading()

  fun hideLoading()

  fun showNoInternet()

  fun showError(message: String)
}
